base/workflow/R/run.write.configs.R

```diff
@@ -22,61 +22,68 @@
 #' @export
 #'
 #' @author David LeBauer, Shawn Serbin, Ryan Kelly, Mike Dietze
-run.write.configs <- function(settings, write = TRUE, ens.sample.method = "uniform", 
-                              posterior.files = rep(NA, length(settings$pfts)), 
+run.write.configs <- function(settings, write = TRUE, ens.sample.method = "uniform",
+                              posterior.files = rep(NA, length(settings$pfts)),
                               overwrite = TRUE) {
-  tryCatch({
-    con <- PEcAn.DB::db.open(settings$database$bety)
-    on.exit(PEcAn.DB::db.close(con), add = TRUE)
-  }, error = function(e) {
-    PEcAn.logger::logger.severe(
-      "Connection requested, but failed to open with the following error: ",
-      conditionMessage(e))
-  })
-  
+  tryCatch(
+    {
+      con <- PEcAn.DB::db.open(settings$database$bety)
+      on.exit(PEcAn.DB::db.close(con), add = TRUE)
+    },
+    error = function(e) {
+      PEcAn.logger::logger.severe(
+        "Connection requested, but failed to open with the following error: ",
+        conditionMessage(e)
+      )
+    }
+  )
+
   ## Which posterior to use?
   for (i in seq_along(settings$pfts)) {
     ## if posterior.files is specified us that
     if (is.na(posterior.files[i])) {
       ## otherwise, check to see if posteriorid exists
       if (!is.null(settings$pfts[[i]]$posteriorid)) {
-        #TODO: sometimes `files` is a 0x0 tibble and other operations with it fail.
+        # TODO: sometimes `files` is a 0x0 tibble and other operations with it fail.
         files <- PEcAn.DB::dbfile.check("Posterior",
-                                        settings$pfts[[i]]$posteriorid, 
-                                        con, settings$host$name, return.all = TRUE)
-        pid <- grep("post.distns.*Rdata", files$file_name)  ## is there a posterior file?
+          settings$pfts[[i]]$posteriorid,
+          con, settings$host$name,
+          return.all = TRUE
+        )
```

Comment:
Revert?

Reply:
I think some of this is diff getting confused (the `pid <- ...` is still there, just shows in the next diff chunk), but IDK why styler didn't move "Posterior" down a line to match. That's a ding on my trust in the  "just run styler::style_file() and it'll be good enough" theory.